library/src/main/java/com/novoda/downloadmanager/RoomDownloadsPersistence.java

```diff
@@ -139,4 +139,46 @@ public boolean update(DownloadBatchId downloadBatchId, boolean notificationSeen)
         database.roomBatchDao().update(roomBatch);
         return true;
     }
+
+    @Override
+    public void persistCompletedBatch(CompletedDownloadBatch completedDownloadBatch) {
```

Comment:
All of the above code was originally exposed inside the `MigrationJob` but this meant client applications were unable to specify there own migration process.